### PR TITLE
PS-5124 : Disable fast index creation option until PS-4892 is impleme…

### PR DESCRIPTION
**Pull request by satya-bodapati**
…nted

Make the variable "expand_fast_index_creation" read-only. Default is OFF
Remove this limitation when PS-4892 is done.

### CODE REVIEW

File: mysql-test/collections/disabled.def [hunk 1]
```diff
@@ -42,6 +42,7 @@ json.json_ddl_ndb : BUG#26616564 FAILURE OF JSON_DDL_NDB TEST, Bug#26609059 2017
 
 # main suite tests
 main.ds_mrr-big @solaris   : BUG#14168107 2012-04-03 Hemant disabled new test added by Olav Sandstå,since this leads to timeout on Solaris on slow sparc servers.
+main.percona_expand_fast_index_creation : PS-4892 2018-12-11 Satya Disabled until expand fast creation is reimplemented
```
Comment (laurynas-biveinis) [inline mysql-test/collections/disabled.def:47]: Please also disable (grep for "expand.fast.index.creation" regex in MTR):
main.bug64663
innodb.percona_expand_fast_index_creation_innodb
sys_vars.expand_fast_index_creation

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: mysql-test/collections/disabled.def (hunk 1 above)
Comment (satya-bodapati) [reply mysql-test/collections/disabled.def:47]: Fixed!

**Review by laurynas-biveinis — APPROVED** [review]

**Comment by peterschwaller** [comment]
touch

**Comment by peterschwaller** [comment]
touch

**Comment by peterschwaller** [comment]
touch

**Pull request merged**